AuthenticationSystem/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views
+
+
+urlpatterns = [
+    path('login_user/', views.login_user, name="login"),
```

Comment:
Remove logout_user url,
rename, login_user to login,
rename, register_user to user